pkg/cmd/roachtest/option/cluster_options.go

```diff
@@ -0,0 +1,143 @@
+// Copyright 2023 The Cockroach Authors.
```

Comment:
Nit: is it already 2024? I always forget :)

Reply:
Do we update copyright headers when files are moved?

Reply:
Nope, I don't think it's common practice to update copyright headers upon renaming. I actually didn't realize it was a rename; at the time of writing, I thought it was a newly added file. 2023 it is!